lib/validLabels.js

```diff
@@ -1,9 +1,19 @@
-export default {
-    bug: 'Bug Fixes',
-    upgrade: 'Dependency Upgrades',
-    documentation: 'Documentation',
-    feature: 'Features',
-    enhancement: 'Enhancements',
-    build: 'Build-Related',
-    breaking: 'Breaking Changes'
-};
+import path from 'path';
+
+let validLabels;
+
+try {
+    validLabels = require(path.join(process.cwd(), 'pr-log.json'));
```

Comment:
Do we still need this code here as we now read the config from `package.json` in `lib/cli.js`?